### PR TITLE
update example

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

File: azure/terraform.tfvars.example [hunk 1]
```diff
@@ -104,3 +97,13 @@ ha_sap_deployment_repo = ""
 # Except salt-minion (for the moment) and salt formulas
 # true or false (default)
 #qa_mode = false
+#
```
Comment (juadk) [inline azure/terraform.tfvars.example]: unwanted line I guess

**Comment by MalloZup** [comment]
thx done. Regarding the devel_mode the idea is to put the docuemtnation in the 1st global readme. 

as we have like the monitoring  https://github.com/SUSE/ha-sap-terraform-deployments

**Comment by juadk** [comment]
> thx done. Regarding the devel_mode the idea is to put the docuemtnation in the 1st global readme. see #168
> 
> as we have like the monitoring https://github.com/SUSE/ha-sap-terraform-deployments

Yes but why not add these new variables in the Azure README? like you did in libvirt README.
I agree it will be better to have a global README but before that, we have to ensure that everything is clear for users. WDYT?

**Comment by MalloZup** [comment]
afaik the libvirt shouldn't have them in readme. only in examples let me check

**Comment by MalloZup** [comment]
https://github.com/SUSE/ha-sap-terraform-deployments/tree/master/libvirt#specifications 
I don't see them in readme.md

**Comment by juadk** [comment]
> https://github.com/SUSE/ha-sap-terraform-deployments/tree/master/libvirt#specifications
> I don't see them in readme.md



> https://github.com/SUSE/ha-sap-terraform-deployments/tree/master/libvirt#specifications
> I don't see them in readme.md

Me I see:
`monitoring_srv_ip: IP address of the machine that will host the monitoring stack`
`devel_mode: Whether or not to install HA/SAP packages from ha_sap_deployment_repo`
Moreover, it lacks `monitored_hosts` in the both

**Comment by MalloZup** [comment]
@juadk  ah yes true. So should we remove it from libvirt or add them to azure? i would prefer to remove them but ok not strong opinion

**Comment by juadk** [comment]
> @juadk ah yes true. So should we remove it from libvirt or add them to azure? i would prefer to remove them but ok not strong opinion

To be honest, as long as we do not have a global README file, I prefer have it in the both.
A new Azure user could be lost if he doesn't see any explanation of these variables in README file.

**Comment by MalloZup** [comment]
@juadk CC . changes adressed tia

File: azure/terraform.tfvars.example [hunk 2]
```diff
@@ -115,3 +108,12 @@ ha_sap_deployment_repo = ""
 # qa_mode must be set to true for executing hwcct
 # true or false (default)
 #hwcct = false
+
```
[the remark this answers is not in the document]
Comment (MalloZup) [reply azure/terraform.tfvars.example]: done

**Comment by MalloZup** [comment]
I messed up the rebase I will need to reopen an pr later

**Pull request closed without merging**